tests/network/user_defined_network/ip_specification/test_ip_specification.py

```
@@ -115,7 +117,12 @@ def test_successful_external_connectivity(self, vm_under_test: BaseVirtualMachin
         assert vm_under_test.console(commands=[f"ping -c 3 {PUBLIC_DNS_SERVER_IP}"], timeout=30)
 
     @pytest.mark.polarion("CNV-12586")
-    def test_seamless_in_cluster_connectivity_is_preserved_over_live_migration(self) -> None:
+    def test_seamless_cluster_connectivity_is_preserved_over_live_migration(
+        self,
+        vm_under_test: BaseVirtualMachine,
```

Ruff (0.14.14)

[warning] Unused method argument: `vm_under_test`

(ARG002)